app/cells/plugins/core/asset_info/show.haml

```diff
@@ -21,7 +24,7 @@
           = number_to_human_size(asset['file_size'])
         %dt Dimensions
         %dd
-          = dimensions
+          = asset['image'] ? dimensions : 'Not Image Asset'
```

Comment:
Wrap the whole `dt`/`dd` Dimensions bit in a conditional and don't render if it's not an image